OrbitQt/orbitmainwindow.cpp

```diff
@@ -64,6 +66,12 @@ OrbitMainWindow::OrbitMainWindow(QApplication* a_App, ApplicationOptions&& optio
   ui->HomeHorizontalSplitter->setSizes(sizes);
   ui->splitter_2->setSizes(sizes);
 
+  QStatusBar* status_bar = new QStatusBar(this);
```

Comment:
I'd prefer to define this in the ui file and not create it in code.

Reply:
I think this is supposed to be created in the code? At least I did not see anything it designer. If it needs to be done in UI which section should this be added to?

Reply:
It's a bit hidden. In the designer: Right-click the main window, select "Create status bar". Then you can access the status bar in code through `OrbitMainWindow::statusBar()`. The Designer then adds a `<widget class="QStatusBar" name="statusBar"/>` directly as child of the QMainWindow.

Reply:
dione.